test/testlib/nuodb_database_utilities.go

```diff
@@ -108,38 +108,46 @@ func StartDatabaseTemplate(t *testing.T, namespaceName string, adminPod string,
 
 	installationStep(t, options, helmChartReleaseName)
 
-	AwaitNrReplicasScheduled(t, namespaceName, tePodNameTemplate, opt.NrTePods)
-	AwaitNrReplicasScheduled(t, namespaceName, smPodName, opt.NrSmPods)
+	if awaitDatabase {
```

Comment:
Sorry for the diff mess. The change done is just to wrap the database await logic in `if awaitDatabase { ... }`.
The change starts here.